### ISSUE TITLE
`get_expert_similarity_loss` is not used

### ISSUE THREAD

**Issue by JPassionQ** [Facico/GOAT-PEFT#3]
You implemented `get_expert_similarity_loss` in the Goat layer. Why was this similarity loss not used during the fine-tuning of any downstream tasks, and only the load balancing loss was retrieved?

**Comment by Facico** [comment]
Sorry for the confusion. `get_expert_similarity_loss` was an intermediate experiment during development. Since it introduces additional hyperparameters, we only did some quick tests and didn’t see a clear improvement, so we didn’t dig deeper into tuning this loss.
